static/panes/tree.ts

```diff
@@ -391,17 +391,20 @@ export class Tree {
 
     private editFile(fileId: number) {
         const file = this.multifileService.getFileByFileId(fileId);
-        if (!file.isOpen) {
-            const dragConfig = this.getConfigForNewEditor(file);
-            file.isOpen = true;
+        // TODO: Ensure this is the proper way. Should an exception be thrown?
```

Comment:
@partouf If no file is found, is this enough?

Reply:
If you check the references for editFile in github you can see where its called from